src/core/annotation.js

```diff
@@ -3768,31 +3781,41 @@ class FreeTextAnnotation extends MarkupAnnotation {
     this.data.annotationType = AnnotationType.FREETEXT;
     this.setDefaultAppearance(params);
     if (this.appearance) {
+      this.data.textContent = "";
```

Comment:
Is this actually correct, since elsewhere in this file `this.data.textContent` contains an Array?
This sort of inconsistency seems like something that could easily lead to bugs (later on) in e.g. the display-layer.